client/lib/domains/utils.js

```diff
@@ -56,6 +56,38 @@ function getTransferStatus( domainFromApi ) {
 	return null;
 }
 
+function getGdprConsentStatus( domainFromApi ) {
```

Comment:
One thing that I notiche here is the discrepancy between `getGdprConsentStatusMessage` and `getGdprConsentStatus`. This one uses ifs and the other uses switch/case. I'd go with switch/case in both places.

Reply:
I thought the same thing, but listened to my personal hobgoblin regarding foolish consistency in this file. :-) Will update. 👍